src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../styles/themes";
+
+.container,
+.overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.container {
+  z-index: 2;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.container[aria-hidden="true"] {
+  display: none;
+}
+
+.overlay {
+  background-color: rgba(41, 47, 76, 0.7);
```

Comment:
any variable you can use?

Reply:
Why we did not use a css variable here? because we needed to add opacity?